.travis.yml

```diff
@@ -190,7 +200,6 @@ _helm_deploy: &helm_deploy
     --set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-security-groups"="${AWS_SG}" \
     --set controller.publishService.enabled=true \
     --wait \
-    --atomic \
```

Comment:
I know you removed this for testing but did you intend to leave it out of forget to add back?

Reply:
removed on purpose